docker/docker_client.go

```diff
@@ -531,6 +534,10 @@ func (c *dockerClient) setupRequestAuth(req *http.Request, extraScope *authScope
 			req.SetBasicAuth(c.username, c.password)
 			return nil
 		case "bearer":
+			if c.registryToken != "" {
+				req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.registryToken))
```

Comment:
Could this be reformatted to share the `req.Header.Set` call with the other ways to obtain a token?

Reply:
Do you mean like https://github.com/containers/image/pull/842/commits/90aca60274d70a498c1b348c9290f2b186a2d559? Or, you meant I should just make `req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.registryToken))` function?

Reply:
I was thinking
```go
var bearerToken string
if c.registryToken != "" {
     bearerToken = c.registryToken
} else {
     // existing code
}
req.Header.Set(…)
```
or maybe
```go
bearerToken := c.registryToken
if bearerToken == "" {
    // existing code
}
req.Header.Set(…)
```

Reply:
I see. I actually tried to avoid a long nested block because that change puts the existing code into `if` statement, but I get your point. I've fixed it.
https://github.com/containers/image/pull/842/commits/ecb74f5c21e52011d7ede140ed34d1f71ee76bfe